### PR TITLE
#588 - octane - guides update - model/index

### PR DESCRIPTION
**Pull request by dennismende**
- Update code samples to make use of new syntax
- Update file paths to be aligned with new project structure

### CODE REVIEW

File: guides/release/models/index.md [hunk 1]
```diff
@@ -106,22 +106,27 @@ lists the drafts for the currently logged in user.
 You might be tempted to make the component responsible for fetching that
 data and storing it:
 
-```javascript {data-filename=app/components/list-of-drafts.js}
-import Component from '@ember/component';
+```javascript {data-file=src/ui/components/list-of-drafts/component.js}
```
Comment (rwjblue) [inline guides/release/models/index.md]: MU was removed from octane (see 
https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html for a more detailed writeup), can you swap the filepath back?

```suggestion
```javascript {data-file=app/components/list-of-drafts.js}
```

File: guides/release/models/index.md [hunk 2]
```diff
@@ -106,22 +106,27 @@ lists the drafts for the currently logged in user.
...
   }
-});
+}
 ```
 
 You could then show the list of drafts in your component's template like
 this:
 
-```handlebars {data-filename=app/templates/components/list-of-drafts.hbs}
+```handlebars {data-filename=src/ui/components/list-of-drafts/template.hbs}
```
Comment (rwjblue) [inline guides/release/models/index.md]: Lets reset this change...

File: guides/release/models/index.md [hunk 3]
```diff
@@ -135,22 +140,27 @@ may want a component to display the number of drafts. You may be
 tempted to copy and paste your existing `willRender` code into the new
 component.
 
-```javascript {data-filename=app/components/drafts-button.js}
-import Component from '@ember/component';
+```javascript {data-file=src/ui/components/drafts-button/component.js}
```
Comment (rwjblue) [inline guides/release/models/index.md]: Same, MU was removed from octane so we need to weak this one.

File: guides/release/models/index.md [hunk 4]
```diff
@@ -135,22 +140,27 @@ may want a component to display the number of drafts. You may be
...
+      this.drafts = data;
     });
   }
-});
+}
 ```
 
-```handlebars {data-filename=app/templates/components/drafts-button.hbs}
-{{#link-to "drafts" tagName="button"}}
+```handlebars {data-filename=src/ui/components/drafts-button/template.hbs}
```
Comment (rwjblue) [inline guides/release/models/index.md]: ⬆️

File: guides/release/models/index.md [hunk 5]
```diff
@@ -135,22 +140,27 @@ may want a component to display the number of drafts. You may be
...
+<LinkTo "drafts" @tagName="button">
```
Comment (rwjblue) [inline guides/release/models/index.md]:
```suggestion
<LinkTo @route="drafts" @tagName="button">
```

File: guides/release/models/index.md [hunk 6]
```diff
@@ -206,7 +216,7 @@ Models define the type of data that will be provided by your server. For
 example, a `Person` model might have a `firstName` attribute that is a
 string, and a `birthday` attribute that is a date:
 
-```javascript {data-filename=app/models/person.js}
+```javascript {data-file=src/models/person.js}
```
Comment (rwjblue) [inline guides/release/models/index.md]: ⬆️

File: guides/release/models/index.md [hunk 7]
```diff
@@ -220,7 +230,7 @@ A model also describes its relationships with other objects. For
 example, an `order` may have many `line-items`, and a
 `line-item` may belong to a particular `order`.
 
-```javascript {data-filename=app/models/order.js}
+```javascript {data-file=src/models/order.js}
```
Comment (rwjblue) [inline guides/release/models/index.md]: ⬆️

File: guides/release/models/index.md [hunk 8]
```diff
@@ -229,7 +239,7 @@ export default class Order extends Model {
 }
 ```
 
-```javascript {data-filename=app/models/line-item.js}
+```javascript {data-filename=src/models/line-item.js}
```
Comment (rwjblue) [inline guides/release/models/index.md]: 😄

**Comment by dennismende** [comment]
Wasn't aware of that. Will change it quickly.

**Comment by dennismende** [comment]
@rwjblue Changed it back now.

In addition I looked at the updated component guides to cross check and paths are updated already everywhere. This should be added as an additional todo in #588

**Pull request merged**